x-pack/plugins/cases/common/utils/user_actions.ts

```diff
@@ -71,3 +72,7 @@ export const isCreateCaseUserAction = (userAction: unknown): userAction is Creat
 
 export const isUserActionType = (field: string): field is UserActionTypes =>
   ActionTypes[field as UserActionTypes] != null;
+
+export const isCategoryUserAction = (userAction: unknown): userAction is CategoryUserAction =>
+  (userAction as CategoryUserAction)?.type === ActionTypes.category &&
+  (userAction as CategoryUserAction)?.payload?.category != null;
```

Comment:
The category can be `null` right? Would it still be a valid category user action if it was null? I think that'd indicate a deletion/removal